compiler/build.gradle

```diff
@@ -23,5 +23,10 @@ tasks.withType(org.jetbrains.kotlin.gradle.tasks.KotlinCompile).configureEach {
   kotlinOptions {
     jvmTarget = JavaVersion.VERSION_1_8
     freeCompilerArgs += "-Xopt-in=kotlin.RequiresOptIn"
+
+    // TODO: remove if check when upgrading to Kotlin 1.4.
```

Comment:
did you mean "remove *the* check .."?

Reply:
Also maybe write an explanation why it's needed?

Reply:
Done